code-challenges.js

```diff
@@ -7,38 +7,73 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+//
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
-
+// For me the first thing that comes to my head is to make a function for problem with conditionals.
+//212 is my baseline for this problem and I have to find three solutions, so I'll be utilizing <, >, =.
+// The return will utilize back-ticks(``) as it'll allow me to intersect my parameter into it.
+// Then I'll repeat this same step slichtly changing it to fit the rest of the problems in this question.
+// When I log these problems I am expecting to get back the results that are expected of me in the question.
+
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+//
+// const boilingPoint = (currentTempt) => {
+//   if(currentTempt < 212){
+//     return `${currentTempt} is below boiling point`
+//   }else if (currentTempt > 212){
+//     return `${currentTempt} is above boiling point`
+//   }else if (currentTempt = 212){
```

Comment:
Great work!  The function is dynamic and addresses all of the points.  Make sure that when evaluating conditions that you are using strict equality operators.  It looks like you are reassigning the value of currentTempt on line 35 to be 212, not necessarily evaluating if it equals 212.